components/docs.css

```diff
@@ -69,4 +86,41 @@ html {
     background-color: lightgray;
     color: var(--primary-blue);
     font-weight: bold;
+}
+
+.open-sidebar,
+.side-bar .close-sidebar {
+    display: none;
+    margin: 2rem;
+    color: var(--secondary-dark-blue);
+    cursor: pointer;
+}
+
+i:hover {
+    color: var(--primary-blue)
+}
+
+@media screen and (max-width: 800px) {
+    .side-bar {
+        display: none;
+        transition: left 1s ease;
+        width: 80vw;
+        height: auto;
+        padding: 1rem;
+        overflow-y: auto;
+        position: fixed;
+        top: 4rem;
+        left: 0;
+        justify-content: center;
+        align-items: center;
+        background-color: var(--white-color);
+        z-index: 888;
```

Comment:
Why is `z-index:888`, I think `z-index: 3` is enough

Reply:
To make sure sidebar in mobile view or smaller screens will always be on top